jekyll/_cci2/credits.md

```diff
@@ -23,7 +23,6 @@ The right plan for you and your team will depend on several factors:
 Let's look at how a Performance plan might use credits. With the Performance plan you have:
 
 - At least one credit block (25,000 credits)
```

Comment:
should be two credit blocks (50,000 credits)

Reply:
fixed